crates/bevy_animation/src/lib.rs

```diff
@@ -115,14 +115,22 @@ impl Default for AnimationPlayer {
 
 impl AnimationPlayer {
     /// Start playing an animation, resetting state of the player
-    pub fn play(&mut self, handle: Handle<AnimationClip>) -> &mut Self {
+    pub fn start(&mut self, handle: Handle<AnimationClip>) -> &mut Self {
         *self = Self {
             animation_clip: handle,
             ..Default::default()
         };
         self
     }
 
+    /// Like `start`, but does nothing if the requested animation is already playing.
```

Comment:
Shouldn't `start` be wrapped in a square brackets to link the method?

Reply:
Correct :)